desktop_version/src/editor.cpp

```diff
@@ -1959,10 +1959,21 @@ bool editorclass::load(std::string& _path)
 
         if (pKey == "script")
         {
+            // load in the linebreak attribute
+            linebreak = "";
+            pElem->QueryStringAttribute("linebreak", &linebreak);
+            char splitvalue = linebreak[0];
```

Comment:
Why only use the first character of `linebreak`?

Reply:
I wanted to use the already existing call to `split`, which took a `char`.

Reply:
You could make the main `split()` take in an `std::string`, and keep the old `char` interface around by using function overloading.